runtime/interpreter/interpreter.go

```diff
@@ -2688,13 +2688,22 @@ var ConverterDeclarations = []ValueConverterDeclaration{
 		nestedVariables: []struct {
 			Name  string
 			Value Value
-		}{{
-			Name: sema.AddressTypeFromBytesFunctionName,
-			Value: NewUnmeteredHostFunctionValue(
-				sema.AddressConversionFunctionType,
-				AddressFromBytes,
-			),
-		}},
+		}{
+			{
+				Name: sema.AddressTypeFromBytesFunctionName,
+				Value: NewUnmeteredHostFunctionValue(
+					sema.AddressConversionFunctionType,
```

Comment:
While at it, rename `AddressConversionFunctionType` to `AddressTypeFromBytesFunctionType`

Reply:
Done